green-labs/rescript-bindings
# `rescript-jest` major update

---
"@greenlabs/rescript-jest": major
"@greenlabs/rescript-testing-library": minor
---

@greenlabs/rescript-jest
* add module ExpectStatic, Describe
* add timeout option at promise, async functions
* change naming using toHaveBeen-
* add argument length functions of 'calledWith'
* apply uncurry at each function
* add mockFactory type
* fix type bug of mockReturnValue
* add expect return type, add expect of resolves, rejects

@greenlabs/rescript-testing-library
* apply jest expect type change

plus) *.bs.js 파일 제거
